concepts/observability-strategy/strategy.md

```diff
@@ -2,11 +2,11 @@
 
 ## Motivation
 
-The Kyma project evolved over time, from an extensibility framework having batteries-included, to a runtime which focusses on providing kubernetes building blocks with enterprise-grade quality. Hereby, the Kyma observability feature was not adapting yet, still providing an out-of-the box in-cluster stack which gets you started immediately, but does not qualify for the enterprise-grade qualities.
+The Kyma project evolved over time, from an extensibility framework with "batteries included", to a runtime that focuses on providing Kubernetes building blocks with enterprise-grade quality. Up until now, the Kyma Observability feature is providing an out-of-the box in-cluster stack that gets you started immediately, but which does not qualify for the enterprise-grade qualities. Various observability backends are already available as managed services from different vendors fulfilling these criterias. The possibility for integration into these offerings will not only bring provide the missing value but also Openness - it is up to the users scenarios which observability system will fit best to the actual requirements.
+When Kyma supports a simple way of instrumenting, collecting and integration of the workloads telemetry data into available observability systems, leveraging the [OpenTelemetry](https://opentelemetry.io/) standards to provide integration options in a vendor-neutral approach, the Kyma Observability feature is ready for it's evolution too.
 
-Observability backends are available in various ways from different vendors as managed services. While users can pick and choose and can leverage benefits like cross-cluster scenarios, the integration aspect into that external managed services is often complex and heterogenous. Kyma should not invest into another opinionated solution for an in-cluster backend with enterprise-grade qualities, it should instead focus on simplifying the instrumentation and collection of the telemetry data and the integration of the data into external solutions. Hereby, it will leverage as much as possible the OpenTelemetry standards to provide integration options in a vendor-neutral approach.
 
-With that Kyma will shift the focus away from providing a full Observability solution, instead it will simplify the instrumentation and integration of telemetry data into external systems.
+With that, Kyma will shift the focus away from providing a full Observability solution; instead it will simplify the instrumentation and integration of telemetry data into external systems and with that will provide the support which is needed by keeping biggest flexibility.
```

Comment:
```suggestion
With that, Kyma will shift the focus away from providing a full Observability solution; instead it will simplify the instrumentation and integration of telemetry data into external systems. Thus, Kyma will keep providing support while at the same time, providing the biggest flexibility.
```